imports/plugins/core/dashboard/client/components/plugin-versions.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+
+export default class PluginVersion extends Component {
+  static propTypes = {
+    versionedPackage: PropTypes.object
```

Comment:
I recommend shape since you rely on the object structure. It should also be required since you don't check for existence.

```js
versionedPackage: PropTypes.shape({
  name: PropTypes.string.isRequired,
  version: PropTypes.string
}).isRequired
```